src/services/dom/types.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const HTTP_HEADERS = {
```

Comment:
The common convention for object members in JavaScript is to use camelCase. Another thing to consider is that these are really enums - I know we're not using TypeScript right now, but the convention there for enum is to use PascalCase (like classes with static members)

Reply:
Good to know 👍 
We use the uppercase convention in the mobile project, and this naturally got applied in Hyperview too, but I'm open to adopting a more widespread accepted convention in Hyperview.